AppCenter/AppCenter/Internals/Storage/MSDBStorage.m

```diff
@@ -72,6 +54,45 @@ - (instancetype)initWithVersion:(NSUInteger)version filename:(NSString *)filenam
   return (self = [self initWithSchema:nil version:version filename:filename]);
 }
 
+- (int)initializeDatabaseWithSchema:(MSDBSchema *)schema version:(NSUInteger)version filename:(NSString *)filename {
```

Comment:
We should be careful on naming a method with `init` or `initialize`. It seems an initializer which isn't actually. @clpolet is a naming expert, any suggestions?

Reply:
`configureDatabaseWithSchema`?